config/webpack.common.js

```diff
@@ -125,7 +125,7 @@ module.exports = function (options) {
        *
        * See: https://webpack.js.org/configuration/resolve/#resolve-modules
        */
-      // modules: [ path.resolve(__dirname, 'src'), path.resolve(__dirname, 'node_modules')]
+      modules: [ helpers.root('src'), helpers.root('node_modules')]
```

Comment:
So let's run with this in dev but not in test :-) I'll set that up, as this really does work for me.